index.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="/styles/styles.css">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="/styles/mobile.css">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+<link href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap" rel="stylesheet">
+<script src="https://kit.fontawesome.com/8a4d4148a1.js" crossorigin="anonymous"></script>
+  <title>Yasmina Mayeh</title>
+</head>
+<body>
+  <div class="page-container">
+    
+
+    <!-- HEADER -->
+    <header>
+      <nav>
```

Comment:
Navigation looks great. Spaced well, good font size, nice CSS effect to show when we are hovering. Page links could be changed to open up the different sections on Yasmin as they are currently empty anchor tags.